app/models/carto/user.rb

```diff
@@ -222,15 +225,48 @@ def google_maps_client_id
   end
 
   # returns a list of basemaps enabled for the user
+  # when google map key is set it gets the basemaps inside the group "GMaps"
+  # if not it get everything else but GMaps in any case GMaps and other groups can work together
+  # this may have change in the future but in any case this method provides a way to abstract what
+  # basemaps are active for the user
+  #
```

Comment:
Seems this comment is no longer valid if you remove the google_maps_enabled? logic below

Reply:
See the end of the function - google maps logic is kept but moved to the end.  I think this comment shows up as an addition to the merge branch because it may have been deleted in the upstream, but it's been in our repo for 3+ years so I kept it: https://github.com/bloomberg/cartodb/blame/blp_dev/app/models/carto/user.rb#L203